engine/sim/sc_profileset.hpp

```diff
@@ -227,6 +243,96 @@ class profile_output_data_t
 
   profile_output_data_t& gear( const std::vector<profile_output_data_item_t>& v )
   { m_gear = v; return *this; }
+
+  double crit_rating() const
```

Comment:
Do you really want all those accessors? Why not just use public class members?

Of course this is the proper way to do it, so if you are comfortable with the maintenance costs and since you already added it, why not.

Reply:
Mainly because I was copy/pasting and wanted to stay consistent with the original code